.github/workflows/pull_request.yml

```
@@ -0,0 +1,51 @@
+name: Test noir-wasm-testing
+
+on:
+  pull_request:
+    branches:
+      - main
```

```suggestion
on:
  pull_request:
    branches:
      - master
```

There is no `main` branch.